<launch>
	<node pkg="turtlebot_example" type="carrot_follower_sim" name="carrot_follower" output="screen">
		<remap from="/carrot" to="/move_base_simple/goal"/>
		<remap from="/robot_pose" to="/gazebo/model_states"/>
		<remap from="/debug_pose" to="/debug/robot_pose"/>
	</node>
	
	<!--
	<node pkg="turtlebot_example" type="ips_converter" name="ips_converter" output="screen">
		<remap from="/indoor_pos" to="/gazebo/model_states"/>
		<remap from="/vo" to="/ips_vo"/>
	</node>
	
	<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
		<param name="output_frame" value="odom"/>
		<param name="freq" value="30.0"/>
		<param name="sensor_timeout" value="1.0"/>
		<param name="odom_used" value="true"/>
		<param name="imu_used" value="false"/>
		<param name="vo_used" value="true"/>
		<param name="debug" value="false"/>
		<param name="self_diagnose" value="false"/>
		<remap from="/odom" to="/odom"/>
		<remap from="/vo" to="/ips_vo"/>
	</node>
	-->

</launch>